calibration/LibriSpeech/README.md

```diff
@@ -0,0 +1,11 @@
+# Calibration set for RNN-T benchmark
+
+These are 500 sequences randomly selected from the **train-clean-100** set in [LibriSpeech](http://www.openslr.org/12/) dataset and to be used as the reference calibration set for MLPerf Inference RNN-T benchmark. 
```

Comment:
Please add that they are under 15 sec, or the processed the same way as the training set.

Reply:
@christ1ne Added another criterium.